### PR TITLE
[Docs] small changes

### PR DESCRIPTION
**Pull request by OskarStark**
- use .yaml instead of .yml
- use App instead of AppBundle namespace
- fixed config paths

### CODE REVIEW

File: docs/reference/custom_action.rst [hunk 1]
```diff
@@ -19,8 +19,7 @@ Add new route to your Admin
 
 .. code-block:: php
 
-    <?php
-    // src/AppBundle/Admin/QuestionnaireAdmin.php
+    // src/App/Admin/QuestionnaireAdmin.php
```
Comment (kunicmarko20) [inline docs/reference/custom_action.rst]: this should be `src/Admin/QuestionnaireAdmin.php`

File: docs/reference/orm.rst [hunk 2]
```diff
@@ -38,7 +38,7 @@ Example using Personal Translation
 .. code-block:: php
 
     <?php
-    // src/AppBundle/Entity/FAQCategory.php
+    // src/App/Entity/FAQCategory.php
```
Comment (kunicmarko20) [inline docs/reference/orm.rst]: same here

File: docs/reference/orm.rst [hunk 3]
```diff
@@ -111,7 +111,7 @@ Example using Personal Translation with Traits
...
```
Comment (kunicmarko20) [inline docs/reference/orm.rst]: same here

File: docs/reference/orm.rst [hunk 4]
```diff
@@ -154,7 +154,7 @@ Example for translation class for Personal Translation
 .. code-block:: php
 
     <?php
-    // src/AppBundle/Entity/FAQCategory/Translation.php
+    // src/App/Entity/FAQCategory/Translation.php
```
Comment (kunicmarko20) [inline docs/reference/orm.rst]: same here

File: docs/reference/orm.rst [hunk 5]
```diff
@@ -227,9 +227,9 @@ Your entities need to explicitly implement getter and setter methods for the knp
 .. code-block:: php
 
     <?php
-    // src/AppBundle/Entity/TranslatableEntity.php
+    // src/App/Entity/TranslatableEntity.php
```
Comment (kunicmarko20) [inline docs/reference/orm.rst]: same here

File: docs/reference/orm.rst [hunk 6]
```diff
@@ -335,9 +335,9 @@ Here is an example:
 .. code-block:: php
 
     <?php
-    // src/AppBundle/Entity/TranslatableEntityTranslation.php
+    // src/App/Entity/TranslatableEntityTranslation.php
```
Comment (kunicmarko20) [inline docs/reference/orm.rst]: same here

**Comment by OskarStark** [comment]
Thanks for the review @kunicmarko20 👍

**Comment by kunicmarko20** [comment]
Thanks @OskarStark

**Pull request merged**